recipes/obake/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.1" %}
+
+package:
+  name: obake
+  version: {{ version }}
+
+source:
+  url: https://github.com/bluescarni/obake/archive/v{{ version }}.tar.gz
+  sha256: 946de817940421d8af8c614c289b74dc6e221ea803fc2b3dacc30e63d7dc3246
+
+build:
+  number: 0
+  run_exports:
+    # SO name changes at minor rev bumps.
+    - {{ pin_subpackage('obake', max_pin='x.x') }}
+  skip: true   # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - clang      # [win]
+    - ninja      # [win]
+  host:
+    - abseil-cpp
+    - boost-cpp
+    - mppp
+    - tbb
+    - tbb-devel
+    - backtrace    # [linux]
+  run:
+    - abseil-cpp
+    - boost-cpp
+    - {{ pin_compatible("mppp") }}
+    - tbb
+    - tbb-devel
+    - backtrace    # [linux]
+
+test:
+  commands:
+    - test -f $PREFIX/include/obake/config.hpp                            # [unix]
+    - if not exist %PREFIX%\\Library\\include\\obake\\config.hpp exit 1   # [win]
+
+about:
+  home: https://bluescarni.github.io/obake/
+  license: MPL2
```

Comment:
Can you use the SPDX identifier, `MPL-2.0`?

Reply:
Ok!